### DISCUSSION TITLE
Unable to create a room and test Hubs project locally

### DISCUSSION THREAD

**Discussion by rishivinod7** [mozilla/hubs#3451] [General Questions & Discussion] [answered]
I can't "create a room" on my localhost server anymore because it gets stuck on the loading objects screen. I can only enter the "Elaborate Vibrant World" room which is running through dev.reticulum.io and seems to run an old instance of my work for some reason. This is a persisting problem even after I have freshly cloned the repo to test once again.

**Comment by rishivinod7** [comment, marked as answer]
It's been fixed, no issues now.

**Reply by lihong-xmuseum** [reply]
How can this problem be repaired? I also have this problem. Could you please tell me how to solve it? Thanks a lot.
